src/rt/trace.d

```diff
@@ -348,31 +328,34 @@ static void trace_times(Symbol* root)
 ///////////////////////////////////
 // Initialize.
 
-static void trace_init()
+private void trace_init()
 {
-    if (!trace_inited)
+    synchronized
```

Comment:
so AFAIK this `synchronized` only protects `gtrace_inited` - a comment would be nice